code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -245,7 +250,8 @@
 		if(!usr.canUseTopic(src, BE_CLOSE, ismonkey(usr)))
 			return
 		usr.visible_message("<span class='notice'>[usr] grabs \the [src.name].</span>", "<span class='notice'>You grab \the [src.name].</span>")
-		var/obj/item/C = new item_chair(loc)
+		var/obj/item/chair/C = new item_chair(loc)
+		C.origin_type = type
```

Comment:
What's this for?

Reply:
This is to avoid having to make new `obj/item/chair` subtypes for every `/obj/structure/chair` coded in that only differs from the parent by an iota (such as the escape bar stool).